stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -1115,6 +1115,10 @@ function(add_swift_target_library_single target name)
     set_target_properties("${target}"
       PROPERTIES
       INSTALL_RPATH "$ORIGIN")
+  elseif("${SWIFTLIB_SINGLE_SDK}" STREQUAL "FREEBSD")
```

Comment:
Would you mind changing L1114 to `elseif(SWIFTLIB_SINGLE_SDK MATCHES "OPENBSD|FREEBSD")` instead?  That would ensure that these stay in sync.

Reply:
I apologize for the delay.

I'm open to the change, but the other similar instances in this file don't tend to group multiple cases together (including others that deal with FreeBSD and OpenBSD), so I was just matching convention.  And there isn't any particular reason that we'd expect any future changes here to be synchronized between FreeBSD and OpenBSD anyway.

OK to leave it as is?

Reply:
Sure, I suppose that should be fine.